.github/workflows/flake8.yml

```diff
@@ -0,0 +1,25 @@
+name: Flake8 Check
+
+on: [pull_request]
+
+jobs:
+  flake8:
+    name: Run Flake8
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: 3.11
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8
+
+    - name: Run Flake8
+      run: flake8
```

Comment:
You could turn this action into a more generic "check package" action and have it call `tox -r -e check`, which includes a call to flake8.

This action would become more generic and its specification would be similar to the CI action, which might have its benefits.

Is this a temporary work around until you can figure out to get flake8 to work correctly with tox?  If so, you might want to say so in this file and what your final intentions would be once that problem is resolved.  Also, you could create an Issue and reference the issue in those comments.